shared/src/commonMain/kotlin/com/bunbeauty/shared/data/mapper/addition/AdditionMapper.kt

```diff
@@ -0,0 +1,22 @@
+package com.bunbeauty.shared.data.mapper.addition
+
+import com.bunbeauty.shared.db.AdditionEntity
+import com.bunbeauty.shared.domain.model.addition.Addition
+
+val mapAdditionEntityToAddition: AdditionEntity?.() -> Addition? = {
```

Comment:
Don't need `?`
Just use `?` outside if you need
Like `additionDao.getAdditionEntity(uuid)?.mapAdditionEntityToAddition()`